addons/mail/models/mail_volume_setting.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MailVolumeSetting(models.Model):
+    """ Represents the volume of the sound that the user of user_setting_id will receive from partner_id. """
+    _name = 'mail.volume.setting'
```

Comment:
Same here: don't use mail namespacing for stuff that do not belong to mail. Probably a ``res.users.settings.volume``, indicating some specialization of ``res.users.settings`` model.

Reply:
The reason for using `mail.` was to follow module guideline (the model being defined here now) but if we are ok to not follow that guideline in this case I'd be more than ok to rename it as suggested.

Reply:
That guideline is a default behavior, generally models are linked to the module. Here is makes more sense to have a res.users (with an "s" hélàs) prefix :) . Helps understanding this is an addition to the user model, done in a separated table for performance purpose.